### ISSUE TITLE
[Feature Request] Add multi-threading to transcoder

### ISSUE THREAD

**Issue by mlapaglia** [karamanolev/WhatManager2#93]
I have an 8 core server, sitting waiting for each file to transcode one at a time takes 8 times longer than it could if all cores were being used.

**Comment by desultory** [comment]
I think you can just start the celery script with more concurrency and it will accomplish this

**Comment by Mechazawa** [comment]
Yes, the celery script already supports this.

**Comment by mlapaglia** [comment]
Right, but the issue comes in with racing other users to complete a transcode. If you are transcoding 1 single album it's only going to transcode one file at a time. With concurrency you can certainly transcode 8 albums at a time though.

I tried implementing it, but python was keeping me from running more python code as a child.
